### PR TITLE
Improving node placement, node UI and adding native dom focus to nodes. 

### PR DESCRIPTION
**Pull request by aasimkhan30**
Now:
image

Before:
image

### CODE REVIEW

File: src/js/handler/mxGraphHandler.js [hunk 1]
```diff
@@ -776,7 +776,7 @@ mxGraphHandler.prototype.getBoundingBox = function(cells)
  * Creates the shape used to draw the preview for the given bounds.
  */
 mxGraphHandler.prototype.createPreviewShape = function(bounds)
-{
+{
```
Comment (lewis-sanchez) [inline src/js/handler/mxGraphHandler.js]: Nit: Get rid of this whitespace change.

**Review by lewis-sanchez — APPROVED** [review]

**Review by kburtram — APPROVED** [review]
LGTM.  There are a bunch of hard-coded sizing and placement values that I guess are based on visual observation.  Are those values going to work generically if the size of icons change, etc.?

**Comment by aasimkhan30** [comment]
@kburtram, currently we are using fixed size for the icons. I will move those values to constants for now so that we can easily configure them later.

**Comment by kburtram** [comment]
@aasimkhan30 makes sense, I see there were hard-coded values in the previous code too, so the change looks fine to me.  isolating the constants might make it easier to tune the UI in the future if the requirements change.

**Pull request merged**